admin.php

```diff
@@ -43,7 +37,30 @@ public function handle()
      */
     public function html()
     {
+        /** @var helper_plugin_acknowledge $helper */
+        $helper = plugin_load('helper', 'acknowledge');
+
+        $acks = $helper->getAcknowledgements();
+
         ptln('<h1>' . $this->getLang('menu') . '</h1>');
+
+        echo '<table>';
+        echo '<tr>';
+        echo '<th>' . $this->getLang('overviewPage') . '</th>';
+        echo '<th>' . $this->getLang('overviewUser') . '</th>';
+        echo '<th>' . $this->getLang('overviewTime') . '</th>';
+        echo '</tr>';
+
+        foreach ($acks as $ack) {
+            echo '<tr>';
+            echo '<td><a href=">' . wl($ack['page']) .
+                '">' . $ack['page'] . '</a></td>' .
+                '<td>' . $ack['user'] . '</td><td>' . dformat($ack['ack']) . '</td>';
```

Comment:
depending on the auth backend, the user name may HTML special chars. Better escape this, and just for good measure, the page as well.

Reply:
Won't escaping the page potentially interfere with generating the link with `html_wikilink()` ?

Reply:
yeah when you use `html_wikilink()` it will do the escaping for you